Examples/MAX78002/SDHC_Raw/main.c

```diff
@@ -296,13 +296,13 @@ int main(void)
     }
 
     /* Configure for fastest possible clock, must not exceed 52 MHz for eMMC */
-    if (SystemCoreClock > 96000000) {
-        printf("SD clock ratio (at card) 4:1\n");
-        MXC_SDHC_Set_Clock_Config(1);
-    } else {
-        printf("SD clock ratio (at card) 2:1\n");
-        MXC_SDHC_Set_Clock_Config(0);
-    }
+    // if (SystemCoreClock > 96000000) {
+    //     printf("SD clock ratio (at card) 4:1\n");
+    //     MXC_SDHC_Set_Clock_Config(1);
+    // } else {
+    //     printf("SD clock ratio (at card) 2:1\n");
+    //     MXC_SDHC_Set_Clock_Config(0);
+    // }
```

Comment:
Are we leaving these in just in case?

Reply:
Thanks, good catch.  I was, but they can be removed now